Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -101,16 +101,16 @@
 # step5 is digi+l1tracktrigger
 # step6 is fastsim
 # step7 is fastsim harvesting
-upgradeSteps=['GenSimFull','GenSimHLBeamSpotFull','DigiFull','RecoFullLocal','RecoFull','RecoFullHGCAL','RecoFullTracking','RecoFullTrackingPU','HARVESTFull','DigiTrkTrigFull','FastSim','HARVESTFast','DigiFullPU','RecoFullPU','RecoFullPUHGCAL','RecoFullPUTracking','HARVESTFullPU','DigiFullTrigger']
+upgradeSteps=['GenSimFull','GenSimHLBeamSpotFull','DigiFull','RecoFullLocal','RecoFull','RecoFullHGCAL','RecoFullTracking','RecoFullTrackingPU','HARVESTFullTracking','HARVESTFullTrackingPU','HARVESTFull','DigiTrkTrigFull','FastSim','HARVESTFast','DigiFullPU','RecoFullPU','RecoFullPUHGCAL','RecoFullPUTracking','HARVESTFullPU','DigiFullTrigger']
```

Comment:
This change conflicts with #14540 (FYI @hengne).

The "RecoFullTracking"/"HARVESTFullTracking" as in this PR are (almost) equivalent to "RecoFull_trackingOnly"/"HARVESTFull_trackingOnly" in #14540. Almost means that
* "RecoFull_trackingOnly" in #14540 includes also MINIAODSIM in `--datatier` and `--eventcontent`
  * unintentionally though, probably as a result of #14469 being merged between
* "HARVESTFullTracking" in this PR does not include `+@trackingOnlyDQM`

@boudoul @ebrondol Are "RecoFullTracking"/"HARVESTFullTracking" expected to stay as they are here (i.e. essentially equivalent to "RecoFull_trackingOnly"/"HARVESTFull_trackingOnly" in #14540), or do you foresee updates in them?

If the two naming conventions could be unified, probably the easiest to merge both and try to clean up afterwards.